modules/vaos/app/models/vaos/appointment_time.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module VAOS
+  # This is a custom type class for ensuring Time is always coerced as UTC
+  class AppointmentTime < Virtus::Attribute
+    def coerce(value)
```

Comment:
The underlying API that we're building against have about 7 different date time formats. This and other custom attributes are in an effort to wrangle those so that our front-end only needs to send 1-2 variations at most.